### PR TITLE
shorten dc caused by annotations

### PR DESCRIPTION
**Pull request by deads2k**
Fixes https://github.com/openshift/origin/issues/7328

### CODE REVIEW

File: pkg/deploy/api/types.go [hunk 1]
```diff
@@ -247,10 +247,10 @@ const (
 // These constants represent the various reasons for cancelling a deployment
 // or for a deployment being placed in a failed state
 const (
-	DeploymentCancelledByUser                 = "The deployment was cancelled by the user"
```
Comment (deads2k) [inline pkg/deploy/api/types.go]: @smarterclayton Did you want this in the annotation or only in `oc status`?

Comment (smarterclayton) [inline pkg/deploy/api/types.go]: Hrm - does the describer / event / other output still look ok?  Changing only in status is acceptable, but if the others are ugly that's a good reason to change.

**Comment by kargakis** [comment]
LGTM

**Comment by smarterclayton** [comment]
[merge]

**Pull request merged**